inc/shortcodes/class-facebook.php

```diff
@@ -21,6 +21,11 @@ public static function get_shortcode_ui_args() {
 
 	public static function setup_actions() {
 		add_action( 'init', 'Shortcake_Bakery\Shortcodes\Facebook::action_init_register_scripts' );
+		add_action( 'shortcode_ui_after_do_shortcode', function( $shortcode ) {
+			if ( false !== stripos( $shortcode, '[facebook' ) ) {
```

Comment:
Rather than hardcoding `[facebook`, can we use the return result of `static::get_shortcode_tag()`, so any filter customizations are applied? This way, we won't need to have some additional code in the theme.

Also, it would be good to have a basic test to ensure our preview markup includes the expected scripts.